src/main/java/org/gradle/profiler/maven/MavenScenarioDefinition.java

```diff
@@ -58,6 +58,19 @@ protected String getExecutableName() {
         }
     }
 
+    @Override
+    protected String getExecutablePathWithoutToolHome(File projectDir) {
+        String wrapperName = OperatingSystem.isWindows()
+            ? "mvnw.cmd"
```

Comment:
❓ Should this be `mvnw.bat`?

Reply:
No, it's `mvnw.cmd`, similar to `mvn.cmd` a few lines above.